src/components/DatePicker/index.js

```diff
@@ -16,31 +17,36 @@ class Datepicker extends React.Component {
     constructor(props) {
         super(props);
 
-        this.raiseDateChange = this.raiseDateChange.bind(this);
+        this.setDate = this.setDate.bind(this);
         this.showDatepicker = this.showDatepicker.bind(this);
-
-        /* We're using uncontrolled input otherwise it wont be possible to
-        * raise change events with a date value - each change will produce a date
-        * and make us reset the text input */
-        this.defaultValue = props.value
-            ? moment(props.value).format(CONST.DATE.MOMENT_FORMAT_STRING)
-            : '';
+        const value = DateUtils.getDateAsText(props.value) || DateUtils.getDateAsText(props.defaultValue) || '';
```

Comment:
How come the default here is an empty string but we default to `CONST.DATE.MOMENT_FORMAT_STRING` in ios and android? I think we should just have an empty string be the default everywhere.

Reply:
https://github.com/Expensify/App/pull/7627#discussion_r826204826 . I had explained the issue here. The placeholder makes it consistent among all the platforms. The string is an empty string because we use a DOM datePicker in web/Desktop and it defaults to a similar placeholder on passing an empty string.